coderedcms/tests/test_urls.py

```diff
@@ -47,6 +47,10 @@ def test_search(self):
         self.assertEqual(response.status_code, 200)
         self.assertNotEqual(response.context['results'], None)
 
+        response = self.client.get("/search/?s=keyword&t=t")
```

Comment:
Nice test. Could you change the hard-coded URL to use the `reverse()` function to generate it instead (as in line 41 above)

Reply:
Okay, I updated it. Does follow mean that the search finds the search item? I set this to follow=False to mimic the item not existing but I don't know if that's the right way to do that or if I misunderstood what follow meant.

Reply:
Follow means if the server returns a redirect (301/302 status code) it will follow the redirect. Otherwise it will stop and give you the 301 status code. Generally it is more useful to follow unless you are specifically testing for a 301/302 redirect.